app/code/Magento/Catalog/Model/Category/DataProvider.php

```diff
@@ -350,6 +350,11 @@ public function getAttributesMeta(Type $entityType)
 
             $meta[$code]['scopeLabel'] = $this->getScopeLabel($attribute);
             $meta[$code]['componentType'] = Field::NAME;
+            
+            if($code == "filter_price_range"){
```

Comment:
Cannot be placed in `Catalog` module.

Reply:
@orlangur can you please explain . why it can not be placed in  `Catalog` module

Reply:
Because this is responsibility of `LayeredNavigation`